### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by elena-vi**
Answering the following questions: 
- why list exists?
- how to implement in green field / brown?
- how to measure success?
- how to use the spreadsheet?
- how often to evaluate?
- how to report as a delivery manager?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,71 @@ In addition to these requirements, we also expect projects to use source control
 
 The full checklist can be seen here: [`PRODUCTIONISATION.md`][link_productionisation].
 
+## Why does this list exists?
+
+We believe that following this list will encourage teams to consistently follow best practices and improve their projects, which would lead to making a greater impact by delivering faster and learning more.
+
+## How apply this to your project
```
Comment (yusufsheiqh) [inline README.md]:
```suggestion
## How to apply this to your project
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,71 @@ In addition to these requirements, we also expect projects to use source control
...
+  1. Copy the check-list up for your project - in a new tab in the [spreadsheet][spreadsheet].
```
Comment (yusufsheiqh) [inline README.md]:
```suggestion
  1. Copy the checklist for your project - new tab in the [spreadsheet][spreadsheet].
```

File: README.md [hunk 3]
```diff
@@ -16,6 +16,71 @@ In addition to these requirements, we also expect projects to use source control
...
+  2. Continuously update the checklist, at least once a month.
+  3. Provide feedback, addition and removal of points to the Chalet.
+
+### Granularity
+  Each project that has a unique set of staff-members, instances and stack, should have its own checklist.
```
Comment (yusufsheiqh) [inline README.md]:
```suggestion
  Each project that has a unique set of staff/team members, instances and stack, should have its own checklist.
```

File: README.md [hunk 4]
```diff
@@ -16,6 +16,71 @@ In addition to these requirements, we also expect projects to use source control
...
+
+### Definition of Done
+  Once you’re comfortable that you have satisfied the criteria, tick the box. If you’re on the fence, don’t.
+
+## How to measure success
+We meas ure success by measuring against the following points
```
Comment (yusufsheiqh) [inline README.md]:
```suggestion
We measure success by considering the following points:
```

File: README.md [hunk 5]
```diff
@@ -16,6 +16,71 @@ In addition to these requirements, we also expect projects to use source control
...
+
+#### When will we review
+  - Monthly p15n check-in during Learn Tech before quarterly update
+
+#### What do we want to see
+  - See >5% improvement per month per application
+  - Positive feedback regarding low level of bugs when interviewing delivery managers
+  - Positive review from customer from delivery managers
+
+### Improving the reliability of applications for users
```
Comment (yusufsheiqh) [inline README.md]: Duplication of lines `37-51` in lines `53-67`. 😮

File: README.md [hunk 6]
```diff
@@ -25,3 +90,4 @@ Attribution-ShareAlike 4.0 International: The licensor permits others to copy, d
 Made Tech 2018 &copy;
```
Comment (yusufsheiqh) [inline README.md]:
```suggestion
Made Tech 2019 &copy;
```

File: README.md (hunk 5 above)
Comment (elena-vi) [reply README.md]: 🤦‍♀

**Pull request merged**